app/src/androidTest/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/splash/SplashScreenTest.kt

```diff
@@ -45,6 +53,7 @@ class SplashScreenTest : BaseAndroidComposeTest() {
 
     @Test
     fun when_splash_screen_start_show_both_background_image_and_logo_image() {
+        every { preferenceManager.getBooleanData(PREF_LOGGED_IN) } returns false
```

Comment:
Lots of repetition here 👀 Why don't we put this line in `setup()` and only set it to `true` when needed? 🤔